src/Components/Leaderboard/Leaderboard.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react"
+import "./Leaderboard.css"
+import axios, { AxiosError } from "axios"
+import { Id } from "../../Types/quiz"
+import { LeaderBoardResponseType, LeaderboardType } from "../../Types/leaderboard"
+import { ServerError } from "../../Types/quizContext"
+import "../../softui.css"
+
+export async function GetLeaderboard(id){
```

Comment:
you can export all server requests in a single file maybe. makes the code more modular. this is fine too if you want everything related to a single thing in one place